File: src/main/java/info/youtown/domain/Employee.java
package info.youtown;

import org.springframework.data.annotation.Id;

import lombok.Data;

@Data
public class Employee {
    @Id
    private String id;
    private String firstName;
    private String lastName;
    private String description;

    private Employee() {}

    public Employee(String firstName, String lastName, String description){
        this.firstName = firstName;
        this.lastName = lastName;
        this.description = description;
    }
}

File: src/main/resources/static/components/Employee.js
import React from 'react';
import {Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn} from 'material-ui/Table';

class Employee extends React.Component{
  render() {
    return (
      <TableRow>
        <TableRowColumn>{this.props.employee.firstName}</TableRowColumn>
        <TableRowColumn>{this.props.employee.lastName}</TableRowColumn>
        <TableRowColumn>{this.props.employee.description}</TableRowColumn>
      </TableRow>
    )
  }
}

export default Employee;
